charts/kong/templates/ingress-class.yaml

```diff
@@ -0,0 +1,16 @@
+{{- if (and .Values.ingressController.enabled (not (eq (include "kong.ingressVersion" .) "extensions/v1beta1"))) -}}
+apiVersion: networking.k8s.io/v1
+kind: IngressClass
+metadata:
+  name: {{ .Values.ingressController.ingressClass }}
+  {{- if .Values.ingressController.ingressClassAnnotations }}
+  annotations:
+  {{- range $key, $value := .Values.ingressController.ingressClassAnnotations }}
+    {{ $key }}: {{ $value | quote }}
+  {{- end }}
+  {{- end }}
+  labels:
+  {{- include "kong.metaLabels" . | nindent 4 }}
+spec:
+  controller: konghq.com/ingress-controller
```

Comment:
```suggestion
  controller: kong/ingress-controller
```

Reply:
See https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.20/#ingressclassspec-v1-networking-k8s-io

This should be in the current form.